docs/requirements.txt

```diff
@@ -10,3 +10,4 @@ readthedocs-sphinx-search
 myst-nb
 jupyter_book
 ipykernel>=6.19.4
+tensorflow
```

Comment:
I think the no-entry symbol shown in the diff on github here is trying to tell us that it's a good idea to add a newline at the end. Apparently it's part of the definition of a line that it ends in a newline.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline